components/eamxx/docs/developer/processes.md

```diff
@@ -15,3 +15,4 @@ TODO: add links to papers/github-repos, and a SMALL description
 * spa: prescribed aerosols, blah blah
 * surface coupling: blah
 * mam: prognostic aerosols, blah blah
+nudging: This process is responsible for nudging the model simulation given a set of files with a target nudged state.
```

Comment:
The markdown syntax needs a `*` here.

Reply:
You can click on the three dots next to the file and select `View file`. Github will render md syntax, so you can see here how the page will look like (links may not work, but formatting will)